modules/denoise/config.py

```diff
@@ -5,7 +5,7 @@ def can_build(env, platform):
     # as doing lightmap generation and denoising on Android or HTML5
     # would be a bit far-fetched.
     desktop_platforms = ["linuxbsd", "osx", "windows"]
-    return env["tools"] and platform in desktop_platforms and env["bits"] == "64"
+    return env["tools"] and platform in desktop_platforms and env["bits"] == "64" and env["arch"] != "arm64"
```

Comment:
@bruvzg Doesn't this make the above comment obsolete? It seems that only x86_64 is supported, not just "64-bit architectures".

Reply:
The version we are using is x86_64 only, newer versions of oneDNN should support arm64 as well (there were some issues with updating it, I do not remember what's exactly).